test/package.json

```diff
@@ -3,11 +3,11 @@
   "version": "1.0.0",
   "description": "",
   "main": ".eslintrc.js",
```

Comment:
I assume this is incorrect.

Reply:
https://github.com/digitalbazaar/mocha-w3c-interop-reporter/pull/10/commits/40d3de233ff19e18d512325ff23a2d3437a95bd3